scripts/create-kubestellar-demo-env.sh

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env bash
+
+# The formal instructions accompaning this script can be found at:
```

Comment:
I am not sure I understand the relationship this text is trying to describe. Those instructions do not reference this script.

Reply:
I going to remove it to reduce confusion.